src/Workspaces/CSharp/Portable/Formatting/FormattingHelpers.cs

```diff
@@ -81,6 +81,9 @@ public static bool IsCloseParenInParameterList(this SyntaxToken token)
             return token.Kind() == SyntaxKind.CloseParenToken && token.Parent.Kind() == SyntaxKind.ParameterList;
         }
 
+        public static bool IsCloseParenInTuple(this SyntaxToken token)
```

Comment:
💡 It would be good to have this clearly distinguish between tuple types and tuple expressions

```suggestion
        public static bool IsCloseParenInTupleExpression(this SyntaxToken token)
```